### PR TITLE
F5.7

### PR DESCRIPTION
**Pull request by himik80**

### CODE REVIEW

File: social-network-ui/src/routes/router.js
Comment (bymooon) [inline social-network-ui/src/routes/router.js:1]: Render element SettingsPage in different route.

**Review by bymooon — CHANGES_REQUESTED** [review]
Render element SettingsPage in different route.
Please, use Formik instead of using states.

**Review by miegami — APPROVED** [review]

**Comment by miegami** [comment]
Please, resolve merge conflicts.

File: social-network-ui/src/components/PasswordChenge/PasswordChange.jsx [hunk 1]
```diff
@@ -0,0 +1,114 @@
+import VisibilityIcon from '@mui/icons-material/Visibility';
```
Comment (miegami) [inline social-network-ui/src/components/PasswordChenge/PasswordChange.jsx]: Rename the folder for this file.

**Review by MtTarik — APPROVED** [review]

**Review by bymooon — APPROVED** [review]

**Pull request merged**